connectors/opa/testutil/helper.go

```diff
@@ -0,0 +1,192 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package testutil
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net"
+	"net/http"
+	"strconv"
+	"testing"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/stretchr/testify/assert"
+
+	pb "github.com/ibm/the-mesh-for-data/pkg/connectors/protobuf"
+	"google.golang.org/grpc"
+)
+
+var EnvValues = map[string]string{
+	"CONNECTION_TIMEOUT":    "120",
+	"CATALOG_CONNECTOR_URL": "localhost:50084",
+	"OPA_SERVER_URL":        "localhost:8282",
+}
+
+func GetEnvironment() (int, string, string) {
+	timeOutInSecs := EnvValues["CONNECTION_TIMEOUT"]
+	timeOutSecs, _ := strconv.Atoi(timeOutInSecs)
+
+	catalogConnectorURL := EnvValues["CATALOG_CONNECTOR_URL"]
+	opaServerUrl := EnvValues["OPA_SERVER_URL"]
+	log.Printf("EnvVariables = %v\n", EnvValues)
+
+	return timeOutSecs, catalogConnectorURL, opaServerUrl
+}
```

Comment:
Why not just use constants directly?

Reply:
having the EnvValues variable gives us flexibility to change these env variables in different testcases (if need be) as shown in example [here](https://github.com/rohithdv/the-mesh-for-data/blob/6bfa712c3ac2450a9f9eba2d4051384f9141b514/connectors/opa/lib/opa_reader_test.go#L43).

Reply:
Seems too complex but I assume it's a preparation for actually taking these values from environment variables.

Reply:
actually we wanted to not depend on env variables for running the test cases. hence we created this struct.